### ISSUE TITLE
Feature pack cannot be shared across multiple projects

### ISSUE THREAD

**Issue by jamesnetherton** [wildfly-extras/fuse-patch#101]
The fuse-patch feature pack depends on a specific version of the WildFly feature pack. This is problematic if this feature pack is to be shared across multiple projects which themselves may have feature packs depending on different WildFly versions.

The end result is non-overlapping paths from one feature pack being combined with those of the other(s).

IMO, we're abusing the original intent of feature packs in this instance. All we really need here is something like a compressed artefact containing the relevant module definitions that can be used in other projects by extracting the contents.

**Comment by tdiesler** [comment]
Blocked by: https://issues.jboss.org/browse/WFBUILD-14

**Comment by tdiesler** [comment]
Won't Fix

https://issues.jboss.org/browse/WFBUILD-14?focusedCommentId=13274346&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13274346
